### ISSUE TITLE
Image.anim_delay and -1 documentation is misleading

### ISSUE THREAD

**Issue by tito** [kivy/kivy#2508]
With a zip/png sequence, i wanted to stop it by default:

```
Image:
    anim_delay: -1
    source: "xxx.zip"
```

And i got this on the console: 
```
[CRITICAL] [Clock       ] Warning, too much iteration done before the next frame. Check your code, or increase the Clock.max_iteration attribute
[CRITICAL] [Clock       ] Warning, too much iteration done before the next frame. Check your code, or increase the Clock.max_iteration attribute
...
```

Either this is a bug, either the documentation is misleading: http://kivy.org/docs/api-kivy.uix.image.html?highlight=anim_delay#kivy.uix.image.Image.anim_delay : 
Delay the animation if the image is sequenced (like an animated gif). If **anim_delay is set to -1, the animation will be stopped**.

**Comment by elParaguayo** [comment]
Is this still the case? I've got an `AsyncImage` loading a GIF file and setting `anim_delay` to -1 gave me the same critical warning. Running v1.9.0

**Comment by akshayaurora** [comment]
@elParaguayo sry i guess I forgot to fix this,  should be fixed now. closed by 60031827c773bf7fe7105228710bcb1568fc207e
